readme.md

```diff
@@ -17,16 +17,41 @@ answer goes here
 Describe the homework submission process.
 
 ```
-answer goes here
+1) Fork the repo from GA's github account.
+    - "Forking" a repo copies the repo into my personal github account
 
+2) From the forked repo, click on "Clone" and copy the SSH address.
+
+3) From the terminal, change the directory to the directory you will be working in.
+
+4) Once in that directory, type the following: git clone "paste SSH address". Press enter.
+
+5) Cd back into the directory and open the contents by typing "atom ."
+
+6) Make changes to the file, save, and type "git status" into the terminal.
+    - git status shows your untracked files
+
+7) Type "git add ." to add the file to the staging area.
+
+8) Type "git status" again, files should now ready to be committed.
+
+9) Type "git commit -m "comment on the code" "
+
+10) Once committed, you need to push the repo back to the github cloud. Type: git push origin master
+    - Type git remote -v to verify
+
+11) Go back to Github and create a pull request
+    - Add comfort, completeness, and any comments necessary.  
 ```
 
 ## Question 3
 
 What is the difference between a fork and a clone?
 
 ```
-answer goes here
+Fork: Copies the repo on a remote maching (aka the github cloud)
+
+Clone: Copies the repo onto a local maching (aka the client aka my pc)
 
 ```
 
```

Comment:
This is super close, but crucially, forking a repo copies a repo from another GitHub account to your GitHub account. I'll accept it anyway though 👍